src/hydra/dataProvider.js

```diff
@@ -179,8 +179,19 @@ export default (
       const containFile = (element) =>
         isPlainObject(element) &&
         Object.values(element).some((value) => value instanceof File);
+      const containsMultipartOverride = (data) => {
+        if (data.formEnctype && data.formEnctype === 'multipart') {
+          delete data.formEnctype;
```

Comment:
Given the name of this function, it should be pure (not mutating `data`).

Reply:
Perfect, tks.

Done.